### PR TITLE
Updates after build

### PR DESCRIPTION
**Pull request by adamgreen**
I made a few updates to the build documentation as I was building my Sawppy. Now that my build is complete, I thought I would roll them up in a PR to get some feedback.

It includes:
* Simple typo fixes.
* A couple of URL fixes.
* Updates to build notes to deal with areas that caused me some confusion in the hopes that they will help others in the future.
* Some trailing whitespace that my code editor removed when I touched other lines within a file.

I look forward to feedback!

### CODE REVIEW

File: docs/Print Differential.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ with brackets to mount to body in [Body Box Mounts](https://cad.onshape.com/docu
 **3D Printing STL Files:**
 * [DiffBrace.stl](../STL/DiffBrace.stl) * 2
 * [DiffEnd.stl](../STL/DiffEnd.stl) * 2
-* [DiffLink.stl](../STL/DiffLink.stl) * 2
+* [DiffLink.stl](../STL/DiffLink.stl) * 2 (Mirror one of them)
```
Comment (Roger-random) [inline docs/Print Differential.md]: What did you encounter that required mirroring them for your build? The part is symmetric and should have been identical mirrored or not. Curious to learn more details about why mirroring was necessary.

Comment (adamgreen) [reply docs/Print Differential.md]: Oops. I meant the DiffEnd part. I can push up a correction.

Comment (Roger-random) [reply docs/Print Differential.md]: Aha, that makes more sense. Yes I agree it's worth calling out DiffEnd is to be mirrored.

**Comment by Roger-random** [comment]
Fantastic! I appreciate you submitting changes based on your own Sawppy building experience. Thanks for making things easier for future Sawppy builders.

**Comment by adamgreen** [comment]
Thanks for the review and merge.

**Pull request merged**